src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -45,20 +45,24 @@
 
     private final RoutingRequest request;
     private final Router router;
-
     /**
-     * Transit service time-zero. Usually midnight before the request dateTime, but NOT on days
-     * witch has DST adjustments.
+     * The transit service time-zero normalized for the current search. All transit times are 
+     * relative to a "time-zero". This enables us to use an integer(small memory footprint). The
+     * times are number for seconds past the {@code transitSearchTimeZero}. In the internal model
+     * all times are stored relative to the {@link org.opentripplanner.model.calendar.ServiceDate},
+     * but to be able to compare trip times for different service days we normalize all times by
+     * calculating an offset. Now all times for the selected trip patterns become relative to the
+     * {@code transitSearchTimeZero}.
```

Comment:
This is good

Reply:
I wish it was a better place to put this documentation, but this variable does not have a proper "home", it is just passed down the call stack. So I put the doc on at the top/root of the call stack.